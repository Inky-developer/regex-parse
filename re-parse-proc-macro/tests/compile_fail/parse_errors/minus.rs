use re_parse_proc_macro::re_parse;

fn main() {
    re_parse!("a-b", "111B222");
}