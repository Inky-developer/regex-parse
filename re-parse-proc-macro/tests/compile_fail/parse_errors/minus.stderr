error: Expected end of input, got '-'
 --> tests/compile_fail/parse_errors/minus.rs:4:15
  |
4 |     re_parse!("a-b", "111B222");
  |               ^^^^^
